app/src/main/java/com/github/sdpteam15/polyevents/model/database/local/room/LocalDatabase.kt

```diff
@@ -1,33 +1,50 @@
 package com.github.sdpteam15.polyevents.model.database.local.room
 
 import android.content.Context
+import android.util.Log
```

Comment:
do we really need log in this class or it's just a or it's just a remnant of debug